src/screens/CardGameScreen/index.js

```diff
@@ -92,6 +95,7 @@ class CardGameScreen extends React.Component {
     this.handleVotes(activeIndex, 0);
     Animated.spring(position, {
       toValue: { x: dx, y: -SCREEN_HEIGHT - 100 },
+      // useNativeDriver: true,
```

Comment:
Why is this commented?

Reply:
Must have commented during testing. We can uncomment this line.